fspiop-api/documents/Tracing v1.0.md

```diff
@@ -0,0 +1,895 @@
+CAgICAgICAgICJwYXJ0eUlkZW50aWZpZXIiOiAiMTIzNDU2Nzg5IiwNCiAgICAgICAgI-
+CAgICJmc3BJZCI6ICJNb2JpbGVNb25leSINCiAgICAgICAgfSwNCiAgICAgI-
+CAgInBlcnNvbmFsSW5mbyI6IHsNCiAgICAgICAgICAgICJjb21wbGV4TmFtZSI6IHsNCiAgICAgICAgI-
+CAgICAgICAiZmlyc3ROYW1lIjogIkhlbnJpayIsDQogICAgICAgICAgICAgICAgImxhc3ROYW1lIjogIk-
+thcmxzc29uIg0KICAgICAgICAgICAgfQ0KICAgICAgICB9DQogICAgfSwNCiAgICAicGF5ZXIi-
+OiB7DQogICAgICAgICJwZXJzb25hbEluZm8iOiB7DQogICAgICAgICAgICAiY29tcGxleE5hbWUi-
+OiB7DQogICAgICAgICAgICAgICAgImZpcnN0TmFtZSI6ICJNYXRzIiwNCiAgICAgICAgICAgICAgI-
+CAibGFzdE5hbWUiOiAiSGFnbWFuIg0KICAgICAgICAgICAgfQ0KICAgICAgICB9LA0KICAgICAgI-
+CAicGFydHlJZEluZm8iOiB7DQogICAgICAgICAgICAicGFydHlJZFR5cGUiOiAiSUJBTiIsDQogICAgI- CAgICAgICAicGFydHlJZGVudGlmaWVyI-
+jogIlNFNDU1MDAwMDAwMDA1ODM5ODI1NzQ2NiIsDQogICAgICAgICAgICAiZnNwSWQiOiAiQmFua05yT25 lIg0KICAgICAgICB9DQogICAgfSwNCiAgICAiYW1vdW50Ijogew0KICAgICAgICAiYW1vdW50IjogIjEw-
+MCIsDQogICAgICAgICJjdXJyZW5jeSI6ICJVU0QiDQogICAgfSwNCiAgICAidHJhbnNhY3Rpb25UeXBlI-
+jogew0KICAgICAgICAic2NlbmFyaW8iOiAiVFJBTlNGRVIiLA0KICAgICAgICAiaW5pdGlhdG9yI-
+jogIlBBWUVSIiwNCiAgICAgICAgImluaXRpYXRvclR5cGUiOiAiQ09OU1VNRVIiDQogICAgfSwNCiAgI-
+CAibm90ZSI6ICJGcm9tIE1hdHMiDQp9DQo\u003d\u003d",
+"condition": "fH9pAYDQbmoZLPbvv3CSW2RfjU4jvM4ApG_fqGnR7Xs" 
+}
+```
+
+4.1.3.3. Transfer response callback from MobileMoney to Switch
+
+MobileMoney receives the request, and mutates the trace information before responding with the **PUT /transfers** callback to the Switch.
+
+**Note:** See previous section [4.2.1. Request Party Information](#421-request-party-information) for more detail.
+
+The following trace headers will be include in the request:
+
+**Note:** The fields highlighted below as **`value`** will indicate a newly generated or modified value depending on context. Non-highlighted fields will indicate an unchanged value.
+
+> traceparent: 00-91e502e28cd723686e9940bd3f378f85-**`da6dbe3af96ed639`**-01<br/>
+> tracestate: **`mobilemoney`**=**`da6dbe3af96ed639`**,moja=bb68d98a0840aa99,banknrone=82f942e2566117c0<br/>
+
+<br/>
+The request sample including trace headers:
+
+```http
+PUT /transfers/11436b17-c690-4a30-8505-42a2c4eafb9d HTTP/1.1
+Content-Type: application/vnd.interoperability.transfers+json;version=1.0
+Content-Length: 166
+Date: Tue, 15 Nov 2017 10:14:02 GMT
+FSPIOP-Source: MobileMoney
+FSPIOP-Destination: BankNrOne
+traceparent: 00-91e502e28cd723686e9940bd3f378f85-da6dbe3af96ed639-01
+tracestate: mobilemoney=da6dbe3af96ed639,moja=bb68d98a0840aa99,banknrone=82f942e2566117c0
+{
+    "fulfilment": "mhPUT9ZAwd-BXLfeSd7-YPh46rBWRNBiTCSWjpku90s",
+    "completedTimestamp": "2017- 11-16T04:15:35.513+01:00",
+    "transferState": "COMMITTED"
+}
+```
+
+4.1.3.4. Transfer response callback from Switch to BankNrOne
+
+Switch receives the request, and mutates the trace information before forwarding the **PUT /transfers** callback to BankNrOne.
+
+**Note:** See previous section [4.2.1. Request Party Information](#421-request-party-information) for more detail.
```

Comment:
The link seems to be incorrect, I would guess that it should be to 4.1.1.2. Request Party Information from Switch to MobileMoney?

Reply:
Fixed. It's meant to refer to "4.1.1. Request Party Information" in general.